src/entities/savingsGoals.js

```diff
@@ -97,6 +97,31 @@ class SavingsGoals {
       headers: defaultHeaders(accessToken)
     });
   }
+
+  /**
+   * Add money to a specific savings goal
+   * @param {string} accessToken - the oauth bearer token.
+   * @param {string} savingsGoalId - the savings goal's ID.
+   * @param {string} transactionId - a transaction ID for this transaction
+   * @param {number} amount - an amount in minor unit
+   * @return {Promise} - the http request promise
+   */
+  addMoneyToSavingsGoal (accessToken, savingsGoalId, transactionId, amount) {
+    typeValidation(arguments, addMoneySavingsGoalParameterDefinition);
+    const url = `${this.options.apiUrl}/api/v1/savings-goals/${savingsGoalId}/add-money/${transactionId}`;
+    log(`PUT ${url}`);
+    return axios({
+      method: 'PUT',
+      url,
+      headers: postHeaders(accessToken),
+      data: JSON.stringify({
+        amount: {
+          currency: 'GBP',
```

Comment:
The currency shouldn't be hardcoded to GBP, but instead there should be a `currency` param. If not present, this should default to GBP.